cookbooks/zarr-stac-report/data-producers/index.qmd

```diff
@@ -0,0 +1,277 @@
+
+## What are your highest priorities?
+
+It's usually not possible to optimize for all benefits at the same time. That's why we recommend picking out a few of your highest priorities. Examples include:
+    
+1. Streamline variable-level discovery for web-based users.
+2. Integrate with existing tools and workflows.
+3. Enable the simplest possible access patterns for users from `<insert-programming-language>`
+4. Enable web-based visualization of large-scale multi-dimensional datasets.
+5. Minimize the number of GET requests for accessing large-scale multi-dimensional datasets.
+6. Minimize data transfer for accessing subsets of large-scale multi-dimensional datasets.
+7. Minimize the amount of infrastructure required to maintain the catalog.
+8. Minimize the cost/time of generating the catalog.
+9. Limit who can read from the dataset without limiting who can inspect the metadata.
+
+For each of these examples, prioritizing one over the other impacts how much effort you dedicate to abstracting metadata into the STAC catalog. Your choices will also impact the number of GET requests needed for common access patterns and the amount of data transferred.
+
+## Options
+
+So given your priorities how should you structure the division between what metadata belongs in STAC and what belongs in Zarr? To a certain extent it depends on the shape of your data. 
+
+### One big Zarr store
+
+If you have aligned data cubes [Level 3 and 4 data][7] or groups of data cubes that tend to be global in scale (for instance CMIP6 data or ERA5) then you will want to use this approach.
+
+![](../../../images/one-big-zarr+stac-diagram.png)
+
+At a minimum this setup looks like:
+
+1. Create a Zarr store for each dataset (i.e., set of data collected using the same platform, algorithms, model, etc)
+2. Define a STAC collection for each Zarr dataset.
+3. In each collection include a collection-level asset containing the href link to the Zarr store.
+
+**Pros**
+
+- There is no metadata duplication so the STAC side is easy to maintain.
+- Simple access interface for Python users - no client-side concatenation.
+
+**Cons**
+
+- Potentially many GETs to construct the data cube if there is no [consolidated metadata][8] file.
+- Data variables are not exposed at the STAC level, so users cannot discover relevant datasets by searching for variables.
+
+### One big Zarr store - some STAC metadata
+
+Same as above but with the additional step:
+
+4. Write variable metadata about the groups up into the STAC collection as collection-level metadata using the [Data Cube Extension][9].
+
+**Pros**
+
+- Data variables are exposed at the STAC level, so users can discover relevant datasets by searching for variables.
+- Simple access interface for Python users - no client-side concatenation.
+
+**Cons**
+
+- Potentially many GETs to construct the data cube if there is no consolidated metadata file.
```

Comment:
This `con` seems like a separate concern from having metadata for a Zarr store in STAC.

Reply:
I think I was imagining the alternative as having all the metadata in STAC and being able to fully construct the data cube with just the STAC metadata. But maybe that's not fair as a con in this scenario and should just be a pro the most metadata scenario. (This is the kind of double-counting problem I _always_ have with pros and cons list)